ci/build-project.ps1

```diff
@@ -15,7 +15,7 @@ param(
 Write-Output "Downloading the testing project from $($test_repo_url)"
 git clone -b "$test_repo_branch" "$test_repo_url" "$test_repo_path" --depth 1
 if (-Not $?) {
-    Throw "Failed to clone testing project from $test_repo_url."
```

Comment:
Thinking about it we probably don't need any of the powershell changes just yet. Even though some of this is broken